src/features/swap/helpers/utils.ts

```diff
@@ -72,6 +69,49 @@ export const getSurplusPrice = (
   }
 }
 
+export const getPartiallyFilledSurplus = (order: SwapOrder): number => {
+  if (order.kind === OrderKind.BUY) {
+    return _getPartiallyFilledBuySurplus(order)
+  } else if (order.kind === OrderKind.SELL) {
+    return _getPartiallyFilledSellSurplus(order)
+  } else {
+    return 0
+  }
+}
+
+const _getPartiallyFilledBuySurplus = (
```

Comment:
@katspaugh do we have a convention for _? If not I would remove it. As this function is not being exported it is clear that it is "private".

Reply:
Yes, no need for _ for not exported functions. The convention is the same as in Python, e.g. if you export something only for unit tests, it can be prefixed with an _.